server/src/services/notification.services.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+import { sendEmail } from "../emailService/sendEmail.js";
+import { getStudentEmailByApplicationId } from "../services/application.services.js";
+
+export const sendNotificationApplicationDecision = async (applicationId, status) => {
+  try {
+    const subject = "Thesis Management - Professor application decision";
+    const htmlMessage = `
+    <html>
+      <head>
+        <style>
+          body {
+            font-family: Arial, sans-serif;
+            background-color: #f4f4f4;
+            color: #333;
+          }
+          .container {
+            width: 80%;
+            heigth: 100vh;
+            margin: auto;
+            padding: 2%;
+            background-color: #fff;
+            border-radius: 5px;
+            box-shadow: 0 2px 5px rgba(0,0,0,0.1);
+          }
+          h1 {
+            color: #007bff;
+          }
+        </style>
+      </head>
+      <body>
+        <div class="container">
+          <h1>Application ${applicationId} Status Update</h1>
```

Comment:
Wouldn't it be more clear if the title of the proposal and the name of the professor were shown? Because the applicationId is just an internal information the users don't really see

Reply:
fixed